packages/reakit/src/Menu/Menu.md

```diff
@@ -0,0 +1,12 @@
+A menu that does nothing for now...
+
+```jsx
+<Menu>
```

Comment:
```suggestion
import { Menu } from "reakit";

<Menu>
```

Reply:
Oh interesting, @diegohaz you know why I didn't see any error when rendering this locally due to the missing `import`?

Reply:
Technically, those imports aren't needed (styleguidist already creates global variables for them). But we started to adopt it as a convention in #276 so as to make it clearer for people reading the docs.

Reply:
Maybe we should find a way to disable that auto import on styleguidist so we can catch missing imports more easily. I'm not sure if it's possible though.